### PR TITLE
Add Safe modules configuration

### PR DESCRIPTION
**Pull request by germartinez**
Check https://github.com/gnosis/contract-proxy-kit/issues/89

The following methods won't be added for now until the new CPKFactory contract is available:
- transfer ETH out of the GnosisSafeProxy: `cpk.transfer(destination, amountInWei)`
- send ETH along with `execTransactions`: `cpk.execTransactions(txs, {value})`

### CODE REVIEW

File: src/CPK.ts [hunk 1]
```diff
@@ -302,6 +316,63 @@ class CPK {
...
+    return modules
+  }
+
+  async isSafeModuleEnabled(moduleAddress: Address): Promise<boolean> {
+    if (!this.#contract) {
+      throw new Error('CPK contract uninitialized')
+    }
+    const modules = await this.#contract.call('getModules', [])
+    const selectedModules = modules.filter(
+      (mod: Address) => mod.toLowerCase() === moduleAddress.toLocaleLowerCase()
```
Comment (cag) [inline src/CPK.ts:333]: An address is a string, so just `toLowerCase()` should be fine? I don't think there's a difference though.

File: src/CPK.ts [hunk 2]
```diff
@@ -302,6 +316,63 @@ class CPK {
...
+      throw new Error('CPK contract uninitialized')
+    }
+    const ownerAccount = await this.getOwnerAccount()
+    if (!ownerAccount) {
+      throw new Error('CPK ownerAccount uninitialized')
+    }
+    const sendOptions = normalizeGasLimit({ ...options, from: ownerAccount })
+    const txResult = await this.#contract.send('enableModule', [moduleAddress], {
+      ...sendOptions,
+      from: ownerAccount
```
Comment (cag) [inline src/CPK.ts:352]: I think `enableModule` follows the rule for `SelfAuthorized`:

```
require(msg.sender == address(this), "Method can only be called from this contract");
```

So essentially you would use `execTransaction` to call it.

**Review by cag — COMMENTED** [review]
Would this be necessary, considering `execTransactions` already makes this possible?

File: src/CPK.ts (hunk 2 above)
Comment (germartinez) [reply src/CPK.ts:352]: Yes, I'm updating this and pushing the tests :)

File: src/CPK.ts (hunk 1 above)
Comment (germartinez) [reply src/CPK.ts:333]: Good catch! It was the VSCode autocomplete. Thank you!

**Review by rmeissner — APPROVED** [review]

File: src/CPK.ts [hunk 3]
```diff
@@ -287,6 +301,73 @@ class CPK {
...
+    if (!isProxyDeployed) {
+      throw new Error('CPK Proxy contract is not deployed')
+    }
+    if (!this.#contract) {
+      throw new Error('CPK contract uninitialized')
+    }
+    return await this.#contract.call('getModules', [])
+  }
+
+  async isModuleEnabled(moduleAddress: Address): Promise<boolean> {
```
Comment (rmeissner) [inline src/CPK.ts:315]: 1.2.0 has a contract method for this ;)

**Pull request merged**